app/build.gradle

```diff
@@ -232,6 +232,7 @@ dependencies {
     implementation "org.mozilla.components:telemetry:${Versions.telemetry}"
 
     implementation "com.adjust.sdk:adjust-android:${Versions.adjust}"
+    implementation "com.android.installreferrer:installreferrer:${Versions.installreferrer}"
```

Comment:
Google Play Install Referrer Library
https://developer.android.com/google/play/installreferrer/library

Reply:
Do we need data review for this? Did this use location services?
This may not be an issue for now, but we may need to keep a note on this..